src/components/accession2/view/Accession2View.tsx

```diff
@@ -452,7 +452,7 @@ export default function Accession2View(): JSX.Element {
               reload={reloadData}
               setNewViabilityTestOpened={setOpenNewViabilityTest}
               changeTab={handleChange}
-              title={accession?.viabilityPercent ? strings.EDIT_VIABILITY : strings.ADD_VIABILITY}
+              title={accession?.viabilityPercent?.toString() ? strings.EDIT_VIABILITY : strings.ADD_VIABILITY}
```

Comment:
Suggestion: use IsNaN instead of toString? And use numbers instead of toString conversions..?

Reply:
Yes, I can use NaN in some places, but not in the value of the Textfield. I need to convert it to "0" so that the textfield shows the value

Reply:
Got it, we should update textfield component to do the right thing. Can we create a ticket for that - the validation should distinguish between text and number? Also we should be using type=number in these text fields (i can't remember why we aren't..).
Looks good for now

Reply:
Yes, I will create the ticket